torch/csrc/distributed/rpc/process_group_agent.h

```diff
@@ -12,9 +12,17 @@ namespace torch {
 namespace distributed {
 namespace rpc {
 
+constexpr auto kDefaultNumSendRecvThreads = 4;
+
 struct ProcessGroupRpcBackendOptions : public RpcBackendOptions {
-  ProcessGroupRpcBackendOptions() = default;
-  int numSendRecvThreads;
+  ProcessGroupRpcBackendOptions(
+      int num_send_recv_threads,
+      std::chrono::milliseconds rpc_timeout,
+      std::string init_method)
+      : RpcBackendOptions(rpc_timeout, init_method),
+        numSendRecvThreads_(num_send_recv_threads) {}
```

Comment:
Would we want to do some error checking here (or higher in Python layer) and throw if an invalid number such as negative is passed in?